deploy/cloudbuild-release-lts.yaml

```diff
@@ -7,52 +7,15 @@ steps:
     args:
     - 'build'
     - '-t'
-    - 'gcr.io/$PROJECT_ID/build_deps:latest-lts'
-    - '--cache-from'
-    - 'gcr.io/k8s-skaffold/build_deps:latest-lts'
-    - '-f'
-    - 'deploy/skaffold/Dockerfile.deps.lts'
-    - '.'
-
-# Grab secret credentials from gcp bucket
-  - name: gcr.io/cloud-builders/gcloud
-    entrypoint: 'bash'
-    args: ['deploy/setup-secret.sh','-p', $PROJECT_ID]
-
-# Build and tag skaffold builder
-  - name: 'gcr.io/cloud-builders/docker'
-    args:
-    - 'build'
-    - '--cache-from'
-    - 'gcr.io/$PROJECT_ID/skaffold-builder:latest'
```

Comment:
I think we need to add 
```yaml
 - '--build-arg'
 - 'SKAFFOLD_VERSION=$TAG_NAME'
```

to inject version to the build context

Reply:
Ah, sorry, I was testing this with a modified version of this file (mostly because I didn't want to push to AR too) and then I forgot to copy my fixes back into this file. Fixed!